api/src/main/java/me/lokka30/treasury/api/economy/misc/EconomyAPIVersion.java

```diff
@@ -11,7 +11,9 @@
  * @since v1.0
  */
 public enum EconomyAPIVersion {
-    v1_0(new short[]{1, 0});
+    v1_0(new short[]{1, 0}),
+    v1_1(new short[]{1, 1}),
```

Comment:
I don't think this should increment the API version. See #140

Reply:
Will address it the next time I'm opening my IDE.

Reply:
Remove it and just dropped referencing Economy API Version there.